packages/next-codemod/bin/upgrade.ts

```diff
@@ -263,3 +286,61 @@ async function suggestCodemods(
 
   return codemods
 }
+
+async function suggestReactCodemods(packageManager: PackageManager) {
+  const { runReactCodemod } = await prompts(
+    {
+      type: 'toggle',
+      name: 'runReactCodemod',
+      message: 'Do you want to run the React 19 upgrade codemod?',
+      initial: true,
+      active: 'Yes',
+      inactive: 'No',
+    },
+    { onCancel }
+  )
+
+  if (runReactCodemod) {
+    const commandMap = {
+      yarn: 'yarn dlx',
+      pnpm: 'pnpx',
+      bun: 'bunx',
+      npm: 'npx',
+    }
+    const command = commandMap[packageManager] || 'npx'
+
+    // https://react.dev/blog/2024/04/25/react-19-upgrade-guide#run-all-react-19-codemods
+    execSync(`${command} codemod@latest react/19/migration-recipe`, {
+      stdio: 'inherit',
+    })
+  }
+}
+
+async function suggestReactTypesCodemods(packageManager: PackageManager) {
+  const { runReactTypesCodemod } = await prompts(
+    {
+      type: 'toggle',
+      name: 'runReactTypesCodemod',
+      message: 'Do you want to run the React Types codemod?',
+      initial: true,
+      active: 'Yes',
+      inactive: 'No',
+    },
+    { onCancel }
+  )
+
+  if (runReactTypesCodemod) {
+    const commandMap = {
+      yarn: 'yarn dlx',
+      pnpm: 'pnpx',
+      bun: 'bunx',
+      npm: 'npx',
+    }
+    const command = commandMap[packageManager] || 'npx'
+
+    // https://react.dev/blog/2024/04/25/react-19-upgrade-guide#typescript-changes
+    execSync(`${command} types-react-codemod@latest preset-19 .`, {
```

Comment:
Feeling inclined to not prompt here 
```suggestion
    execSync(`${command} types-react-codemod@latest --yes preset-19 .`, {
```

If people want such a custom upgrade story, they should not use the upgrade codemod.